src/tools/dev/scripts/bvauto.py

```diff
@@ -90,7 +90,7 @@ def get_svn_root():
     """
     Constructs a svn root for the VisIt repo using env var SVN_NERSC_NAME.
     """
-    if not "SVN_NERSC_NAME" in os.environ.keys():
+    if not "SVN_NERSC_NAME" in list(os.environ.keys()):
```

Comment:
Wouldn't this be...
```
if "SVN_NERSC_NAME" not in os.environ:
```

Reply:
yes it could also be, however I 2to3 outputs are a safe bet, I wanted to limit hand editing

Reply:
this is another script we can remove, masonry replaces it